gnovm/tests/files/recover12b.gno

```diff
@@ -0,0 +1,28 @@
+package main
+
+func anotherRecover() {
+	if r := recover(); r != nil {
+		println(r)
+	}
+}
+
+func main() {
+	defer func() {
+		if r := recover(); r != nil {
+			println(r + ": not another recover")
```

Comment:
This doesn't compile in go:

`invalid operation: r + ": not another recover" (mismatched types interface{} and string)`

Yet an other gno/go difference!

Reply:
```suggestion
			println(r.(string) + ": not another recover")
```

Reply:
Updated as you suggested to future proof it; I just verified the issue will be fixed when #1426 is merged.